src/libsyncengine/update_detection/update_detector/updatetreeworker.h

```diff
@@ -107,8 +107,6 @@ class UpdateTreeWorker : public ISyncWorker {
 
         ExitCode createMoveNodes(const NodeType &nodeType);
 
-        [[nodiscard]] bool updateNodeId(std::shared_ptr<Node> node, const NodeId &newId);
```

Comment:
You can keep it this way: 

```
[[nodiscard]] bool updateNodeId(std::shared_ptr<Node> node, const NodeId &newId) { return _updateTree->updateNodeId(node, newId); }
```

Reply:
Not sure I see the advantage of this.  
From my point of view, these kinds of functions that just forward the arguments make the class heavier than necessary and make navigating through the codebase less efficient. For example, searching for all usages of `UpdateTree::updateNodeId()` means first searching for `UpdateTree::updateNodeId()`, then searching for all functions that simply forward the arguments, such as `UpdateTreeWorker::updateNodeId()`, and so on.

Reply:
It would have made your changes more concise, with the potential to ease off further refactoring. Resolving.